.github/workflows/test-e2e-gov.yml

```diff
@@ -52,7 +52,11 @@ jobs:
           cd test/e2e
           ginkgo labels
           echo 'Running: AKO_E2E_TEST=1 ginkgo --label-filter="atlas-gov" --timeout 120m --nodes=10  --flake-attempts=1 --randomize-all --race --cover --v --trace --show-nodes-events --output-interceptor-mode=none' && \
-          AKO_E2E_TEST=1 ginkgo --label-filter="atlas-gov" --timeout 120m --nodes=10 --flake-attempts=1 --randomize-all --race --cover --v --trace --show-node-events --output-interceptor-mode=none --coverpkg=github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/...
+          AKO_E2E_TEST=1 ginkgo --label-filter="atlas-gov" --timeout 120m --nodes=10 --flake-attempts=1 --randomize-all --race --cover --v --trace --show-node-events --output-interceptor-mode=none --coverpkg=github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/... | tee .e2e.log
+          if grep -q "0 Passed" < .e2e.log ; then
+            echo "No tests were run!"
+            exit 1
```

Comment:
I am a bit concerned with the repetition of this line across many scripts below.

Reply:
The CI should be running the same code as the local dev, but it does not. That is why this fix needs so much repetition.

Fixing the duplicity is a bigger issue than the focus of this PR. It has already been flagged before but not prioritised.

Reply:
@josvazg I took a quick look on Ginkgo and we can reduce duplication by adding the following code to the Ginkgo suites we have.

```
var _ = ReportAfterSuite("Ensure test suite was not empty", func(r Report) {
	Expect(r.SuiteSucceeded && r.PreRunStats.SpecsThatWillRun > 0).To(BeTrue(), "Suite must run at least 1 test in order to be valid")
})
```

What do you think?

Reply:
Uhm, yes I think that would be better.

There is a problem, when running locally we are calling the `int` tests and `int/clusterwide` somewhat separately, so we might run a label that only runs on one of them and the other should not complain. There is probably another solution for it, but need to figure that out.

In the meantime, I need to understand the `helm-update` test failure, which only happens in the CI for some reason.

Reply:
Applied this suggestion already.